cumulusci/tests/test_schema.py

```diff
@@ -9,7 +9,7 @@
 
 class TestSchema:
     @pytest.mark.skipif(
-        sys.version_info < (3, 7), reason="requires python3.7 or higher"
```

Comment:
@Br4nd0R you can remove this skipif since the tests will always be running in py3.7+ now.